### DISCUSSION TITLE
login issue

### DISCUSSION THREAD

**Discussion by roycheled** [milo526/homebridge-tuya-web#277] [Q&A] [answered]
Hello All. 

for some reason I can't login to my tuya account  with this plugin. 
gives me an error. 
am I the only one with this issue?

**Comment by milo526** [comment]
Please use the search functionality in this repository.
There is a wiki entry regarding this problem, there have been multiple discussions, and there is information in issues.

If this still does not help you, provide at least some information that I can use to help you; "It doesn't work" gives me nothing to work off of.

**Comment by roycheled** [comment, marked as answer]
Thanks for your reply.
That is exactly what I did, and everything works fine!

All the best,

Roy
